etc/grafana/dashboards/overview.json

```diff
@@ -9736,9 +9736,9 @@
             "uid": "${DS_PROMETHEUS}"
           },
           "editorMode": "code",
-          "expr": "reth_blockchain_tree_make_canonical_duration{action=\"clone old blocks\", quantile=\"0.5\"} ",
+          "expr": "reth_blockchain_tree_make_canonical_duration{instance=~\"$instance\", action=~\"$blockchain_tree_action\", quantile=\"0.5\"} ",
```

Comment:
let's leave only `0.5`, `0.95` and `1` percentile values, so we would get an idea of median, near-maximum and maximum values without too many lines on the chart